marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -85,8 +87,10 @@ import Web.UIEvent.KeyboardEvent.EventTypes (keyup)
 initialState :: State
 initialState =
   State
-    { view: Simulation
+    { view: Simulation {- TODO: I think we should change the type to Maybe and the initial state should be Nothing-}
     , jsCompilationResult: NotCompiled
+    {- QUESTION: Are these being used? from what I gathered blockly and actusBlockly are actual halogen components
+    so they handle their own state -}
     , blocklyState: Nothing
     , actusBlocklyState: Nothing
```

Comment:
the halogen component part is only the wrapping up of the javascript, they also have their own "normal" component stuff.

Reply:
After clearing up the question in a meeting we decided to remove this from the state as it wasn't being used